plugins/remap_stats/CMakeLists.txt

```diff
@@ -17,5 +17,4 @@
 
 add_atsplugin(remap_stats remap_stats.cc)
 
-target_link_libraries(remap_stats PRIVATE ts::tscore)
```

Comment:
It seems `tscore/ink_config.h` is included by remap_stats.cc but is not actually needed. We can maybe remove it. It also includes `tscore/ink_defs.h` for the `unlikely` definition; should we duplicate the definition in the plugin to avoid the dependency?

Reply:
I say here "not actually needed", but "is needed and a feature flag is missing that should be there" is another possibility.

Reply:
unfortunately lots of plugins include stuff from tscore, but they still cannot link tscore.  Removing includes is fine if they aren't used, but outside of scope for this PR.